File: app/src/main/java/com/example/happyenglish/VhodViewModel.java
package com.example.happyenglish;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.AndroidViewModel;
import androidx.lifecycle.LiveData;

import com.example.happyenglish.data.MainRepository;

public class VhodViewModel extends AndroidViewModel {
    private final MainRepository repository = new MainRepository(getApplication());


    public LiveData<Boolean> loginV(String login, String pass) {
        return repository.login(login, pass);
    }

    public VhodViewModel(@NonNull Application application) {
        super(application);
    }
}

File: app/src/main/java/com/example/happyenglish/VhodViewModel.kt
package com.example.happyenglish

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import androidx.lifecycle.LiveData
import com.example.happyenglish.data.MainRepository

class VhodViewModel(application: Application) : AndroidViewModel(application) {
    private val repository = MainRepository(getApplication())


    fun loginV(login: String?, pass: String?): LiveData<Boolean> {
        return repository.login(login, pass)
    }
}
